src/main/java/frc/robot/config/FluidConstant.java

```diff
@@ -76,14 +76,6 @@ public T getValue() {
         return this.value;
     }
     
-    /**
-     * Get method for implementing the Supplier interface
-     * @return The current value of this constant
-     */
-    public T get() {

```

Comment:
Any reason that this was removed?

Reply:
I removed this since it was a duplicate that was somehow created during a merge.